<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Good Games</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lbl_alf">
         <property name="text">
          <string>Алфавит</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cor_ord">
         <property name="text">
          <string>Буквы в правильном порядке</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_vow_con">
         <property name="text">
          <string>Гласные и согласные буквы</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lbl_sch">
         <property name="text">
          <string>Счёт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ASMD">
         <property name="text">
          <string>Арифметика</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_compare">
         <property name="text">
          <string>Сравнение</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lbl_mir">
         <property name="text">
          <string>Окружающий мир</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nutr">
         <property name="text">
          <string>Животные по типу питания</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="lbl_login">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_balls">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_logout">
         <property name="text">
          <string>Выйти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_login">
         <property name="text">
          <string>Войти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_register">
         <property name="text">
          <string>Зарегистрироваться</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
